foundational-cxx-support-matrix.md

```diff
@@ -3,31 +3,31 @@
 | Dimension       | Supported Version      | Last Changed | Next Change [^next-change] |
 |-----------------|------------------------|--------------|-------------|
 | C++ Version     | >= 14                  | 2022-07-01   | 2024-12-15  |
-| CMake           | >= 3.13                | 2023-06-01   | 2024-07-01 [^cmake] |
+| CMake           | >= 3.18                | 2024-07-01   | 2024-11-22 [^cmake] |
 | Bazel           | 6 LTS                  | 2023-04-18   | 2024-11-01  |
 | GCC             | >= 7.5.0               | 2024-07-01   | 2026-01-01 [^gcc] |
 | Clang           | >= 7.0.0               | 2023-06-01   | 2025-04-02 [^clang] |
 | MSVC            | >= 2022                | 2024-04-29   | 2027-01-12  |
 | Apple Clang     | >= 12                  | 2022-07-01   | |
 | Android NDK API | == 21                  | 2022-09-18   | |
-| Alpine          | >= 3.16                | 2024-01-02   | 2024-06-01 |
-| Debian          | >= 10                  | 2022-07-01   | 2024-07-01 |
+| Alpine          | >= 3.17                | 2024-07-01   | 2024-11-22 |
+| Debian          | >= 11                  | 2024-07-01   | 2026-06-30 |
 | Fedora          | >= 39                  | 2024-01-02   | 2024-12-01 |
 | openSUSE        | >= Leap 15.5           | 2024-01-02   | 2025-01-01 |
 | Ubuntu LTS      | >= 20.04               | 2023-06-01   | 2025-04-02 |
 | RHEL            | >= 9                   | 2024-07-01   | 2027-06-01 |
 | RockyLinux      | >= 9                   | 2024-07-01   | 2027-06-01 |
 | Windows Server  | >= 2022                | 2024-01-22   | 2026-10-13 |
 | Windows Client  | >= 10                  | 2022-07-01   | 2025-11-01 |
-| macOS           | >= 10.15 (Catalina)    | 2024-01-24   | 2024-07-01 |
+| macOS           | >= 12.7 (Monterey)     | 2024-07-01   | 2024-10-01 |
```

Comment:
I think there are internal teams (Chrome?) that require us to support older versions, even though Apple no longer does.  Does not make sense to me either.  Consider separating this to its own PR so we can merge the other stuff.